deltachat-ios/Controller/ProfileInfoViewController.swift

```diff
@@ -4,6 +4,8 @@ import DcCore
 class ProfileInfoViewController: UITableViewController {
 
     weak var coordinator: EditSettingsCoordinator?
+    var onClose: VoidFunction?
```

Comment:
i would prefer a name that does not look like a system-event here, maybe sth. as onProfileInfoDone.

Reply:
no need to block this pr, though, just for the future, it is better to be a bit more precise.